addons/l10n_sa_edi/tests/test_edi_zatca.py

```diff
@@ -32,6 +33,60 @@ def testInvoiceStandard(self):
 
             self.assertXmlTreeEqual(current_tree, expected_tree)
 
+    def testInvoiceWithDownpayment(self):
+
+        if self.env['ir.module.module']._get('sale').state != 'installed':
```

Comment:
nitpick:
Even better to use `_installed` to avoid a `SELECT` on `ir.module.module`